lib/components/card.js

```diff
@@ -78,19 +78,18 @@ export default {
     props,
     render(h, { props, data, slots }) {
         let childNodes = [],
-            variantPrefix = props.bordered ? "border-" : "bg-";
-
-        // The order of these conditionals matter.
-        // We are building up the markup.
-        if (slots().img) {
-            childNodes.push(slots().img);
-        } else if (props.img && !props.imgBottom) {
-            childNodes.push(
+            variantPrefix = props.bordered ? "border-" : "bg-",
+            img =
+                slots().img ||
                 h(CardImg, {
                     props: { top: props.imgTop, bottom: props.imgBottom, imgFluid: props.imgFluid },
                     attrs: { src: props.img, alt: props.imgAlt }
-                })
-            );
+                });
+
+        // The order of these conditionals matter.
+        // We are building up the markup.
+        if (props.imgTop || !props.imgBottom) {
```

Comment:
I shouldn't need to check `props.img` because of this:
```js
let img =
    slots().img ||
    h(CardImg, {
        props: { top: props.imgTop, bottom: props.imgBottom, imgFluid: props.imgFluid },
        attrs: { src: props.img, alt: props.imgAlt }
    });
```
And yes, img-top if specified, or if bottom is not specified.